src/ape/options.py

```diff
@@ -31,3 +36,35 @@ def verbose_option(help=""):
         default=False,
         help=help,
     )
+
+
+class Alias(click.Choice):
+    """Wraps ``click.Choice`` to load account aliases for the active project at runtime."""
+
+    name = "alias"
+
+    def __init__(self, account_type: Type[AccountAPI] = KeyfileAccount):
+        # NOTE: we purposely skip the constructor of `Choice`
+        self.case_sensitive = False
+        self._account_type = account_type
+
+    @property
+    def choices(self) -> List[str]:  # type: ignore
+        # NOTE: This is a hack to lazy-load the aliases so CLI invocation works properly
```

Comment:
I am not sure if this comment is still needed...
CLI invocation seems to work fine and this does not seem all that "lazy" as it did before